webviz_subsurface/plugins/_group_tree/_views/_group_tree_view/_view.py

```diff
@@ -0,0 +1,350 @@
+from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple
+
+import webviz_core_components as wcc
+import webviz_subsurface_components as wsc
+from dash import Input, Output, State, callback, html
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC, ViewABC
+
+from ..._types import NodeType, StatOptions, TreeModeOptions
+from ..._utils import EnsembleGroupTreeData
+from ._view_element import GroupTreeViewElement
+
+
+class ViewControls(SettingsGroupABC):
+    class Ids(str, Enum):
+        TOUR_STEP = "tour-step"
+        ENSEMBLE = "ensemble"
+        TREEMODE = "tree-mode"
+
+    def __init__(self, ensembles: List[str]) -> None:
+        super().__init__("Controls")
+        self._ensembles = ensembles
+
+    def layout(self) -> html.Div:
+        return html.Div(
+            id=self.register_component_unique_id(self.Ids.TOUR_STEP),
+            children=[
+                wcc.Dropdown(
+                    id=self.register_component_unique_id(self.Ids.ENSEMBLE),
```

Comment:
Strictly seen, you would need to use `self.Ids.ENSEMBLE.value` here (and all other places). However, the plugin will still work if you don't, it just adds another `Ids` in front of your actual literal. In https://github.com/equinor/webviz-config/pull/619 I have implemented a new subclass of `Enum` called `StrEnum`. With this one, no `.value` is required anymore. We will probably globally replace all `(str, Enum)` with `StrEnum` when it is is merged. So don't worry about this, just as an info :wink: